features/step_six.feature

```diff
@@ -7,19 +7,25 @@ Feature: Step six page
 
   Scenario: Displays step number
     Then I should see step 6 of 20
-  
+
   Scenario: Displays header
     Then I should see 'Do you receive any of the following benefits?' header
 
   Scenario: Displays benefits list
```

Comment:
Maybe call this something like 'Displays instruction bullet points' or 'Displays instructions'